content/ko/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -14,49 +14,99 @@ card:
-* 컴퓨터의 특정 포트들 개방. 자세한 내용은 [여기](#check-required-ports)를 참고한다.
-* 스왑의 비활성화. kubelet이 제대로 작동하게 하려면 **반드시** 스왑을 사용하지 않도록 설정한다.
+{{< note >}}
+`kubeadm` 설치는 동적 링킹을 사용하는 바이너리를 통해 수행되며 대상 시스템이 `glibc`를 제공한다고 가정한다.
+이는 많은 리눅스 배포판(데비안, 우분투, 페도라, CentOS 등 포함)에서 합리적인 가정이지만
+알파인 리눅스(Alpine Linux)와 같이 기본적으로 `glibc`를 포함하지 않는 커스텀 및 경량 배포판에서는 항상 그런 것은 아니다.
+배포판이 `glibc`를 포함하거나 예상되는 심볼을 제공하는
+[호환성 레이어](https://wiki.alpinelinux.org/wiki/Running_glibc_programs)를 포함할 것으로 예상된다.
+{{< /note >}}
 
+<!-- steps -->
 
+## OS 버전 확인
 
-<!-- steps -->
+{{% thirdparty-content %}}
+
+{{< tabs name="operating_system_version_check" >}}
+{{% tab name="Linux" %}}
+
+* kubeadm 프로젝트는 LTS 커널을 지원한다. [LTS 커널 목록](https://www.kernel.org/category/releases.html)을 참고한다.
+* `uname -r` 명령을 사용하여 커널 버전을 확인할 수 있다
+
+자세한 정보는 [리눅스 커널 요구사항](/docs/reference/node/kernel-version-requirements/)을 참고한다.
 
-## MAC 주소 및 product_uuid가 모든 노드에 대해 고유한지 확인 {#verify-mac-address}
+{{% /tab %}}
+
+{{% tab name="Windows" %}}
+
+* kubeadm 프로젝트는 최신 커널 버전을 지원한다. 최신 커널 목록은 [윈도우 서버 릴리스 정보](https://learn.microsoft.com/en-us/windows/release-health/windows-server-release-info)를 참고한다.
+* `systeminfo` 명령을 사용하여 커널 버전(OS 버전이라고도 함)을 확인할 수 있다
+
+자세한 정보는 [윈도우 OS 버전 호환성](/docs/concepts/windows/intro/#windows-os-version-support)을 참고한다.
+
+{{% /tab %}}
+{{< /tabs >}}
 
-* 사용자는 `ip link` 또는 `ifconfig -a` 명령을 사용하여 네트워크 인터페이스의 MAC 주소를 확인할 수 있다.
-* product_uuid는 `sudo cat /sys/class/dmi/id/product_uuid` 명령을 사용하여 확인할 수 있다.
+kubeadm으로 생성된 쿠버네티스 클러스터는 커널 기능을 사용하는 소프트웨어에 의존한다.
+이 소프트웨어에는 {{< glossary_tooltip text="컨테이너 런타임" term_id="container-runtime" >}},
+{{< glossary_tooltip term_id="kubelet" text="kubelet">}}, 그리고 {{< glossary_tooltip text="컨테이너 네트워크 인터페이스" term_id="cni" >}} 플러그인이 포함되지만 이에 국한되지 않는다.
 
-일부 가상 머신은 동일한 값을 가질 수 있지만 하드웨어 장치는 고유한 주소를 가질
-가능성이 높다. 쿠버네티스는 이러한 값을 사용하여 클러스터의 노드를 고유하게 식별한다.
-이러한 값이 각 노드에 고유하지 않으면 설치 프로세스가
-[실패](https://github.com/kubernetes/kubeadm/issues/31)할 수 있다.  
+지원되지 않는 커널 버전으로 인한 예기치 않은 오류를 방지하기 위해, kubeadm은 `SystemVerification` 사전 검사를 실행한다. 이 검사는 커널 버전이 지원되지 않으면 실패한다.
+
+kubeadm이 커널 버전을 지원하지 않더라도, 커널이 필요한 기능을 제공한다는 것을 알고 있다면 검사를 건너뛰도록 선택할 수 있다.
+
+## 모든 노드에서 MAC 주소와 product_uuid가 고유한지 확인 {#verify-mac-address}
+
+* `ip link` 또는 `ifconfig -a` 명령을 사용하여 네트워크 인터페이스의 MAC 주소를 확인할 수 있다
+* `sudo cat /sys/class/dmi/id/product_uuid` 명령을 사용하여 product_uuid를 확인할 수 있다
+
+하드웨어 장치는 고유한 주소를 가질 가능성이 매우 높지만, 일부 가상 머신은 동일한 값을 가질 수 있다. 쿠버네티스는 이러한 값을 사용하여 클러스터의 노드를 고유하게 식별한다.
+이러한 값이 각 노드에서 고유하지 않으면 설치 과정이 [실패](https://github.com/kubernetes/kubeadm/issues/31)할 수 있다.
 
 ## 네트워크 어댑터 확인
 
 네트워크 어댑터가 두 개 이상이고, 쿠버네티스 컴포넌트가 디폴트 라우트(default route)에서 도달할 수 없는
 경우, 쿠버네티스 클러스터 주소가 적절한 어댑터를 통해 이동하도록 IP 경로를 추가하는 것이 좋다.
 
 ## 필수 포트 확인 {#check-required-ports}
-[필수 포트들](/ko/docs/reference/networking/ports-and-protocols/)은
-쿠버네티스 컴포넌트들이 서로 통신하기 위해서 열려 있어야
-한다. 다음과 같이 [netcat](https://netcat.sourceforge.net)과 같은 도구를 이용하여 포트가 열려 있는지 확인해 볼 수 있다.
+
+쿠버네티스 컴포넌트가 서로 통신하려면 이러한 [필수 포트](/docs/reference/networking/ports-and-protocols/)가 열려 있어야 한다.
```

Comment:
There is Korean version - please change to refer to https://kubernetes.io/ko/docs/reference/networking/ports-and-protocols/